.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+on:
+  push:
+    tags:
+    - '*'
+    branches:
+    - master
+    - release-*
+  pull_request:
+    types:
+    - opened
+    - synchronize
+jobs:
+  ui:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.49
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Create combined mono repo lock file
+      uses: ./.github/actions/create-concatenated-ui-monorepo-lock
+
+    - name: Cache UI dependencies
+      uses: ./.github/actions/cache-ui-dependencies
+
+    - name: Fetch UI deps
+      run: make -C ui deps
```

Comment:
You could drop this because `make -C ui test` depends on `deps`. Optional and up to you.